src/main/java/uk/gov/hmcts/reform/darts/controllers/authentication/AuthInternalUserController.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.darts.controllers.authentication;
+
+import groovy.util.logging.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+@Slf4j
+@RestController
+@RequestMapping("/internalUser")
+public class AuthInternalUserController extends AuthController {
+
+    @GetMapping("/login-or-refresh")
```

Comment:
Do you think we should be consistent with the url names? For example, you have camel case for the request mapping part `/internalUser` and then kebab case for the get mapping part `/login-or-refresh`

Reply:
good point. I will update the mapping url to use underscore.